src/chunkserver/chunkserver-common/cmr_disk.h

```diff
@@ -129,4 +137,7 @@ class CmrDisk : public FDDisk {
 	/// Writes to device custom blockSize from blockBuffer
 	int writeChunkData(IChunk *chunk, uint8_t *blockBuffer, int32_t blockSize,
 	                   off64_t offset) override;
+
+private:
+	ChunkTrashManager trashManager;
```

Comment:
All functions in ChunkTrashManager are static, you probably don't need an instance.